modules/ROOT/pages/index.adoc

```diff
@@ -15,8 +15,10 @@ The Anypoint Platform Private Cloud Edition uses Docker and Kubernetes to provid
 
 == See Also
 
-* xref:prereq-workflow.adoc[Workflow: Install and Verify Prerequisites for Physical Servers]
+* xref:supported-cluster-config.adoc[Supported Network Topologies]
```

Comment:
Replaced repeated link with a link to cluster configuration / network topologies.